cmd/snap/cmd_list.go

```diff
@@ -76,9 +76,11 @@ func listSnaps(names []string) error {
 
 	for _, snap := range snaps {
 		notes := &Notes{
-			Private: snap.Private,
-			DevMode: snap.DevMode,
-			TryMode: snap.TryMode,
+			Confinement: snap.Confinement,
```

Comment:
We need JailMode here rather than Confinement, as we may have it on even if we have a properly confined snap and --jailmode.